pmoves/docs/audit/CI_ENFORCEMENT_2026-08-10.md

```
@@ -0,0 +1,435 @@
+$ find . -name 'test_*.py' -not -path '*/venv/*' -not -path '*node_modules*' | wc -l
+264
+$ find . -name 'test_*.py' -not -path '*/venv/*' -not -path '*node_modules*' | head -20 | wc -l
+20
+```
+
+244 of 264 test files are never executed, and *which* 20 run is filesystem
+order — not a stable, reviewable subset.
+
+**Discarded status.** A deliberately failing test, run through the exact step:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)